api/src/utils/get-snapshot.ts

```diff
@@ -26,12 +26,12 @@ export async function getSnapshot(options?: { database?: Knex; schema?: SchemaOv
 	]);
 
 	const collectionsFiltered = collectionsRaw.filter((item: any) => excludeSystem(item));
-	const fieldsFiltered = fieldsRaw.filter((item: any) => excludeSystem(item)).map(omitID) as SnapshotField[];
-	const relationsFiltered = relationsRaw.filter((item: any) => excludeSystem(item)).map(omitID) as SnapshotRelation[];
+	const fieldsFiltered = fieldsRaw.filter((item: any) => excludeSystem(item)) as SnapshotField[];
+	const relationsFiltered = relationsRaw.filter((item: any) => excludeSystem(item)) as SnapshotRelation[];
```

Comment:
Assertion no longer required due to relocation of `map`
```suggestion
	const fieldsFiltered = fieldsRaw.filter((item: any) => excludeSystem(item));
	const relationsFiltered = relationsRaw.filter((item: any) => excludeSystem(item));
```